### PR TITLE
made changes to the English interface

### LINKED ISSUE #95: Make the English translations more natural
Some of the UI in English isn't very natural for a English speaker so need to change certain phrases.

### PR DESCRIPTION
**Pull request by shaimayshah**
Closes #95 
Made changes to the locales/en/common.json file to make the English sound a bit more natural.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,8 @@
 # misc
 .DS_Store
 *.pem
+yarn.lock
```
Comment (SamuraiT) [inline .gitignore:21]: you don't need to ignore yarn.lock and actually it shouldn't be

File: .gitignore [hunk 2]
```diff
@@ -18,6 +18,8 @@
...
+package-lock.json
```
Comment (SamuraiT) [inline .gitignore:22]: do we have this ? them remove cuz we don't use npm

File: locales/en/common.json [hunk 3]
```diff
@@ -5,42 +5,42 @@
...
   "MINT_YOUR_KAMON_MINT_BUTTON": "Mint for 1000 $HENKAKU",
-  "MINT_YOUR_KAMON_DISABLED_BUTTON_HOLDER": "You already hold a Kamon NFT",
+  "MINT_YOUR_KAMON_DISABLED_BUTTON_HOLDER": "You already own a Kamon NFT",
   "CURRENT_HOLDERS_1": "🎉 ",
   "CURRENT_HOLDERS_2": " members have minted so far 🎉",
 
-  "ENABLE_EXPLANATION": "Enable Kamon NFT to access your $HENKAKU, in order to be able to mint.",
-  "ENABLE_BUTTON": "Enable $HENKAKU payments",
+  "ENABLE_EXPLANATION": "Please allow the website to access your $HENKAKU balance",
+  "ENABLE_BUTTON": "Allow",
```
Comment (SamuraiT) [inline locales/en/common.json:28]: hmm, enable/approve is really common in crypto. we don't much use allow in this context

Comment (shaimayshah) [reply locales/en/common.json:28]: Ok let me fix that then. 
Would "Enable the website to access your $HENKAKU balance" be better?

Comment (SamuraiT) [reply locales/en/common.json:28]: usually people don't know actually what the approve/enable means tho

**Pull request closed without merging**